conanfile.py

```diff
@@ -53,4 +54,15 @@ def build(self):
             self.cmake.test()
 
     def package(self):
-        self.cmake.install()
```

Comment:
Avoid CMake install step so we don't get the CMake module overriding the Conan generated module.